### DISCUSSION TITLE
Switching between Audible accounts.

### DISCUSSION THREAD

**Discussion by dazzyp** [openaudible/openaudible#716] [Q&A] [answered]
Hi.
Are there any plans to support audible account switching or multi account logins?

I've purchased a licence with the intention of backing up the purchased content in mine and my wife's separate audible accounts. 
From what i can tell to switch between our two accounts will require logging in and out (and dealing with the login authentication emails, or installing a separate version of the application for each user. 

Just wondering if there is an easier way i've missed.

Thanks!

**Comment by openaudible** [comment]
Currently we don't have an easier way. Some basic support for multiple account management has been put into the code, but we haven't done anything on the UI to enable it. We're trying to keep things simple and switching accounts and libraries is tricky to get right. 

Known feature request!

**Comment by dazzyp** [comment]
Hi
Thanks for getting back to me.
I appreicate htta account switch is not an easy thing to do.

If it makes it any easier, my ask/suggestion is the ability to switch 'source' accounts (whilst keeping the same local library' and not to have a separate repository per user.

All the best and keep up the excellent work!

darren
________________________________

**Comment by BlwAvg** [comment]
I am in the same boat here. Wish there were a way to switch accounts. Or maybe and easy way to swap between credentials.json files.

**Comment by Nathalie52** [comment]
I had OpenAudible years ago when it was free and when I had an Audible account in the US. I no longer have a membership with Audible.com since we're now living in France. Today, I paid to install OpenAudible, but it doesn't sync to my Audible.fr. The existing books in the now paid account are the books I had with Audible.com before (which no longer existed) I didn't realise that the region showed in the account is still US (don't know why that is when my Mac's region is France). Please show me the way to change the region and to sync OpenAudible with my now Audible.fr account. Thank you.

**Comment by openaudible** [comment]
You should be able to log out. Then connect to audible and specify the region in the dropdown menu. Make sure you have your current username and password. If the username is not an email, click the legacy checkbox.      ---- On Sat, 03 Feb 2024 08:51:54 -0800  ***@***.******@***.***> wrote ----  
I had OpenAudible years ago when it was free and when I had an Audible account in the US. I no longer have a membership with Audible.com since we're now living in France. Today, I paid to install OpenAudible, but it doesn't sync to my Audible.fr. The existing books in the now paid account are the books I had with Audible.com before (which no longer existed) I didn't realise that the region showed in the account is still US (don't know why that is when my laptop's region is France). Please show me the way to change the region and to sync OpenAudible with my now Audible.fr account. Thank you.

**Comment by Nathalie52** [comment]
Thank you very much for your help.

**Comment by Nathalie52** [comment]
May I ask another question (I started a new discussion but no one
answered)?

I made a mistake of converting aax to m4b instead of mp3 before deleting
the original files. The now converted books can’t be played in Music (the
replacement for the no longer existed iTunes). Is it possible to re-convert
m4b to mp3 on OpenAudible? Thank you.

**Reply by openaudible** [reply]
We don't get notified of new discussions so get behind on responding.
Yes, you can convert m4b to mp3 in the newer versions of OpenAudible. Select book: Action Menu: Convert to MP3.

**Comment by Nathalie52** [comment]
I did that several times already and just did it again now, it didn't
(still doesn't) work.

**Reply by openaudible** [reply]
On mine, it works perfectly.. I select a "green" book with an M4B but no MP3
Action: Convert to MP3. The Audiobook Info panel: state changes to "Converting ... "

It will take a few minutes. The state will change to: Converterd: m4b, mp3.
Then in the Action Reveal submenu, you should see Reveal xyz.mp3.

Look at the File: Log Window for any error messages. Feel free to copy and paste. 

You may have a corrupt M4B file?

Also check for the latest version, 4.0.3.

**Comment by Nathalie52** [comment]
I have 6 m4b files in OpenAudible right now. I highlight one at a time,
click on Action, then Convert to mp3. The result is a quick flash of
Converting: “0 of 1” and State: (on the right hand side) shows the same
“Converted OK m4b”. No error message. And yes, version 4.0.3

All 6 files have the same result.

**Reply by openaudible** [reply]
That is strange. Nothing gets logged to the File Debug Window?

Upload one of the m4b files to this url https://openaudible.org/check and we can scan it for possible errors and or try to reproduce it on our end.

**Reply by openaudible** [reply]
That file is corrupt. It doesn't play on VLC. When you try to convert it, you'll see an error in the Log Window. You'll need to log into your audible account to get them again or find a new source copy.

**Comment by Nathalie52** [comment]
Thank you. I just uploaded and submitted. That's the first of the 6 files,
titled The Unusual Second Life of Thomas Weaver by Shawn Inmon.

**Comment by Nathalie52** [comment]
Thank you. Audible withdrew those books before the year ended, that must be
the reason.

Thank you.

**Comment by openaudible** [comment, marked as answer]
Support for multiple accounts has been added to the latest beta and will be in the next full release.
https://openaudible.org/beta

**Reply by shaekin** [reply]
I love this feature!  I tried to add another account yesterday, and after entering username and password, I'm taken to an Amazon page that doesn't exist.  Any help would be appreciated!
![image](https://github.com/user-attachments/assets/931ea216-f29d-494d-87a8-adda586bfb54)

**Comment by openaudible** [comment]
Make sure you’re using the latest version. 

Then these tips usually help: https://openaudible.org/docs#login_errors

**Reply by shaekin** [reply]
I just updated again, went through all of the tips, and non of them helped.  Is there a limit to how many accounts you can add? I have two, and was trying to add my third.
